assets/terraform-modules/packet/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -76,9 +76,11 @@ systemd:
       enable: true
       contents: |
         [Unit]
-        Description=Flatcar Metadata Agent
+        Description=Flatcar Container Linux Metadata Agent
         [Service]
         Type=oneshot
+        Restart=on-failure
+        RestartSec=5s
```

Comment:
I suggest we use `10s` as upstream does: https://github.com/kinvolk/coreos-overlay/pull/768/commits/069f3ded1103136a6f121d5e93c9c051a3e5dc25.